Agreements/Internal/5_36.txt

```diff
@@ -0,0 +1,6 @@
+COPX:EGCC:*:ITJON:*:*:London S5:London S36:31000:*:^ITJON
```

Comment:
Better to use LAMAT so this agreement displays slightly beyond the sector boundary, in case of later handoff? 
![image](https://user-images.githubusercontent.com/12017284/82542902-ae288e00-9b4a-11ea-9b10-6209efd55a91.png)

Reply:
```suggestion
COPX:EGCC:*:LAMAT:*:*:London S5:London S36:31000:*:^LAMAT
```